web/src/containers/admin/EditAccount.js

```diff
@@ -55,14 +92,14 @@ class EditAccount extends Component {
             label="Name"
             name="name"
             value={name || ''}
-            onChange={this.handleEditAccount}
+            onChange={changeUserName}
```

Comment:
These are all so much clearer than the previous version! Imagine, actually being able to know what something is going to do!